Python/formatter_unicode.c

```diff
@@ -529,7 +529,7 @@ calc_number_widths(NumberFieldWidths *spec, Py_ssize_t n_prefix,
     /* min_width can go negative, that's okay. format->width == -1 means
        we don't care. */
     if (format->fill_char == '0' && format->align == '=')
-        spec->n_min_width = format->width - n_non_digit_non_padding;
+        spec->n_min_width = format->width + n_non_digit_non_padding;
```

Comment:
What's going on here? Did you intend to commit this?

Reply:
Oops, not at all. Thanks for catching that! That was from my lazy use of git add --all. Reverted and fixed.